src/Templates/TemplateSolutions/CustomTypeEditor/CustomControlLibrary.Design.Client/TypeEditor/CustomTypeEditorViewModelClient.cs

```diff
@@ -0,0 +1,93 @@
+﻿using CustomControlLibrary.ClientServerCommunication;
+using CustomControlLibrary.ClientServerCommunication.DataTransport;
+using CustomControlLibrary.ClientServerCommunication.Endpoints;
+using Microsoft.DotNet.DesignTools.Client;
+using Microsoft.DotNet.DesignTools.Client.Proxies;
+using Microsoft.DotNet.DesignTools.Client.Views;
+using System;
+using System.Diagnostics;
```

Comment:
In all of our repos (including the templates) we have different namespace ordering with the System namespace being at the top. Again, I'm assuming, the repo is lacking a correct .editorconfig.

💭  It is also worth considering use of implicit usings (and file scoped namespaces). This can be done later.

Reply:
These are templates for projects which end up in our customer's domain, with their own editor config.
I'll leave it by VS' default, so, no changes here.